homeassistant/components/ps4/media_player.py

```diff
@@ -126,6 +128,7 @@ def __init__(self, name, host, region, ps4, games_file):
     async def async_added_to_hass(self):
         """Subscribe PS4 events."""
         self.hass.data[PS4_DATA].devices.append(self)
+        await self.async_update_entity_data()
```

Comment:
Shouldn't we only call this when migrating the config entry? If it is version specific we should connect this update to the config entry version.

Reply:
Well if nothing changes in the config entry should we be using migrate_entry?

Reply:
@balloob?